app/Http/Controllers/Import/ImportFailureController.php

```
@@ -0,0 +1,514 @@
+        // Ensure all failures belong to this import
+        $failures = $import->failures()->whereIn('id', $failureIds)->get();
+
+        if ($failures->count() !== count($failureIds)) {
+            return response()->json(['error' => 'Some failures not found in this import'], 404);
+        }
+
+        $user = Auth::user();
+        $updated = 0;
+
+        foreach ($failures as $failure) {
+            $success = match ($action) {
```

PHPStan (2.1.15)

Match expression does not handle remaining value: mixed


(match.unhandled)